src/components/WebsiteTabs/index.js

```diff
@@ -0,0 +1,2 @@
+import WebsiteTabs from './WebsiteTabs';
+export default WebsiteTabs;
```

Comment:
Just wanted to see if you are interested in [`export default from` syntax](https://esdiscuss.org/topic/re-export-default#content-2).

Reply:
@asudoh , thanks for this thought! right now, this component is consistent with other components on the website. however, if/when we add this fully into the `website-addons` package, it might make sense to comb through all the `index.js` files and switch over to this that you describe! i'll make a note or issue about this